### DISCUSSION TITLE
Connect via Console?

### DISCUSSION THREAD

**Discussion by Powerbless** [shizunge/endlessh-go#126] [General]
Hello,

How can i Connect to the Console and execute the endlessh Script with Options/Arguments?

I running endlessh-go in portainer/docker and when i click "Console" and try to Connect to the Console(other Container this way Works) i get nothing.

Can anyone give me a Tipp how can i running the Script with Options to Setup grafana and Prometheus?

Thank you

**Comment by shizunge** [comment]
The container is built from [`gcr.io/distroless/base`](https://github.com/shizunge/endlessh-go/blob/7b8ab54f666c68a86ca3f196b88e07492373a59e/Dockerfile#L9C6-L9C28) 
I think there is no shell in the base image.

The entry point is set to the `/endlessh` executable. You can pass additional arguments of `endlessh` as options to the [`docker run`](https://docs.docker.com/reference/cli/docker/container/run/) command or the equivalent in *portainer*.
